adam-apis/src/main/scala/org/bdgenomics/adam/apis/java/JavaADAMContext.scala

```diff
@@ -51,9 +51,9 @@ class JavaADAMContext(val ac: ADAMContext) extends Serializable {
    *
    * Loads path names ending in:
    * * .bam/.cram/.sam as BAM/CRAM/SAM format,
-   * * .fa/.fa.gz/.fa.bz2/.fasta/.fasta.gz/.fasta.bz2 as FASTA format,
-   * * .fq/.fq.gz/.fq.bz2/.fastq/.fastq.gz/.fastq.bz2 as FASTQ format, and
-   * * .ifq/.ifq.gz/.ifq.bz2 as interleaved FASTQ format.
+   * * .fa/.fasta as FASTA format,
+   * * .fq/.fastq as FASTQ format, and
+   * * .ifq as interleaved FASTQ format.
```

Comment:
Should mention that we read the configured compression codecs from the Hadoop config for this, which by default include .gz and .bz2, but can include more.